### PR TITLE
Python 3.5 compatibility

### PR DESCRIPTION
**Pull request by kansch**

### CODE REVIEW

File: schedule/models/events.py [hunk 1]
```diff
@@ -461,11 +461,8 @@ def __unicode__(self):
             'end': self.end,
         }
 
-    def __cmp__(self, other):
-        rank = cmp(self.start, other.start)
-        if rank == 0:
-            return cmp(self.end, other.end)
-        return rank
+    def __lt__(self, other):
+        return self.start < other.start
```
Comment (carxwol) [inline schedule/models/events.py]: it's not isofunctionnal without end comparison ?

**Review by carxwol — CHANGES_REQUESTED** [review]

File: schedule/models/events.py (hunk 1 above)
Comment (kansch) [reply schedule/models/events.py]:
```
if self.start < other.start or (self.start == other.start and self.end < other.end):
    return True
return False
```

Better?

Comment (carxwol) [reply schedule/models/events.py]: yes, but __cmp__ in python 2 used to combine __lt__, __eq__ and __gt__

do you know where's the comparison between events take place to be sure that we don't have to implement __eq__ and __gt__ ?

Comment (kansch) [reply schedule/models/events.py]: Fixed, commit --amend and push -f

**Review by carxwol — APPROVED** [review]

**Pull request merged**